INTEGRATE.md

```diff
@@ -0,0 +1,32 @@
+# Local integration between Porta/System with Keycloak through Zync
+Install these 3 components to have a complete integration of managed SSO applications in Porta.
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+You can have a keycloak running locally with `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+
+Finally [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
+
+## Run Zync
+From the folder where you have Zync installed, run: `ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -p 5000` so you will get zync running in the port 5000.
+
+## Run Porta
+If your porta is already running, stop it, and if you have `spring` running, also do `spring stop`.
+You will need these configurations:
+  - In `config/rolling_updates.yml`, make sure you have `apicast_v2: true` and `apicast_oidc: true`.
+  - In `config/sandbox_proxy.yml`, make sure you have `apicast_registry_url: <%= ENV.fetch('APICAST_REGISTRY_URL', '<Insert a URL that contains policies in JSON. It can be am empty JSON>') %>`
```

Comment:
OK this is porta issue.
I think by default we should not need this for running porta in development mode. Now that we have custom policies. Opening an issue for that?

Reply:
Better, yes, I will open an issue for that in Jira, but this is definitely out of scope for merging this 😛

Reply:
@hallelujah Created in [THREESCALE-2060](https://issues.jboss.org/browse/THREESCALE-2060) 😄